docs/docs/transactions.md

```diff
@@ -84,11 +91,13 @@ sequelize.transaction(function (t1) {
 });
 ```
 
+### Nested/Partial transactions
```

Comment:
hmm..... I thought it was nested since t2 is inside of t1. but what would be a better header?

Cause you are introducing a new concept here, that one could have several transactions going on, and have some of them not "transactionalised". Even though you are right to say it doesn't fall with CLS. Then again it's a new noteworthy concept that could be grossed over without headers.